lang/python/wiredtiger/init.py

```diff
@@ -47,6 +52,32 @@
 me = sys.modules[__name__]
 sys.path.append(os.path.dirname(__file__))
 
+# Find the sanitizer environment variable.
+# FIXME-WT-13237: Rename these testutil flags
+if os.environ.get("TESTUTIL_TSAN") == "1":
+    import subprocess
+
+    # FIXME-WT-13143 We assume TSan is only compatible with clang here. This may change in the future.
+    command = "clang --print-file-name libtsan.so.0"
+    find_tsan_so = subprocess.run(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+    tsan_so_path = find_tsan_so.stdout.strip()
+    if not os.path.isfile(tsan_so_path):
+        print("Cannot find tsan lib")
+        exit(1)
+
+    ld_preload = os.environ.get("LD_PRELOAD")
+    if not ld_preload:
+        os.environ["LD_PRELOAD"] = tsan_so_path
+        restart_python()
+    elif tsan_so_path not in os.environ["LD_PRELOAD"]:
+        os.environ["LD_PRELOAD"] += f":{tsan_so_path}"
+        restart_python()
+    else:
+        # Python already has TSan linking, but if python calls ./wt in a subprocess this breaks ./wt
```

Comment:
Does "breaks ./wt" here mean the wt utility would fail with the TSan LD_PRELOAD setting (and why)?

Reply:
It causes `./wt` to hang until we timeout. It's is something we've also seen with MSan and command line tools in the past. 
I'd need to hunt down the ticket but from memory it produces a call loop where the instrumented code calls a libc function that in turn calls the instrumented code

Reply:
Thanks for the explanation. It's good to know.